src/simulator_core/entities/assets/asset_defaults.py

```diff
@@ -52,8 +52,10 @@ class PipeDefaults:
 PROPERTY_HEAT_DEMAND = "heat_demand"
 PROPERTY_TEMPERATURE_SUPPLY = "temperature_supply"
 PROPERTY_TEMPERATURE_RETURN = "temperature_return"
+PROPERTY_TEMPERATURE = "temperature"
 PROPERTY_PRESSURE_SUPPLY = "pressure_supply"
 PROPERTY_PRESSURE_RETURN = "pressure_return"
+PROPERTY_PRESSURE = "pressure"
```

Comment:
no, they are still used for the controller

Reply:
Shouldn't the controller link to ports as well?

Reply:
that anyhows will be interested since we need to distinguish based on name, ESDL does not support other option. But here we can introduce prim_supply, sec_supply when needed

Reply:
Made an issue for this #137